web/src/beta/features/Editor/tabs/story/PageIndicator/index.tsx

```diff
@@ -1,4 +1,4 @@
-import { FC } from "react";
```

Comment:
Fixed this because the animation is weird when the number of pages increases by more than 1 at a time.